packages/hop-node/src/db/TransferRootsDb.ts

```diff
@@ -378,6 +405,18 @@ class TransferRootsDb extends BaseDb {
     })
   }
 
+  async getBondedTransferRootsFromTwoWeeks (): Promise<TransferRoot[]> {
+    await this.tilReady()
+    const fromUnix = Math.floor((Date.now() - (OneWeekMs * 2)) / 1000)
+    const filter: KeyFilter = {
+      gte: `${fromUnix}`
```

Comment:
Do we need to add the prefix here along with the `fromUnix` value, like [this](https://github.com/hop-protocol/hop/blob/feature/341/packages/hop-node/src/db/TransferRootsDb.ts#L372)?

Reply:
The prefix is not necessary in this case because the db is already namespaced by using a sub db (each sub db already requires a prefix in the constructor). The prefix on the other sub dbs are also unnecessary but can't be removed without doing a migration